crates/biome_service/CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Biome Service
+
+The Biome Service is where we implement the [Workspace](src/workspace.rs).
+
+The workspace is where Biome keeps all internal state of projects, such as open
+documents, but also more advanced service data, such as the instances of our
+[project layout](../biome_project_layout/) and our
+[dependency graph](../biome_dependency_graph/).
+
+Note that there are two implementations of the `Workspace` trait:
+
+* [`WorkspaceServer`](src/workspace/server.rs) maintains the state itself, and
+  is used both inside the daemon as well as in the CLI when it runs in
+  daemonless mode.
+* [`WorkspaceClient`](src/workspace/client.rs) is used for creating connections
+  to the daemon and communicating with the `WorkspaceServer`.
+
+## Watcher
+
+The state inside our workspace is kept in sync with the file system using the
+[`WorkspaceWatcher`](src/workspace_watcher.rs). The watcher is only active in
+daemon mode and not used by the CLI.
+
+### Debugging
+
+Debugging the watcher can be tricky, because you need to run the daemon,
+interact with the file system, and observe the daemon's state somehow.
+
+Debugging is possible with the VS Code extension, but for an easier experience,
```

Comment:
```suggestion
Debugging is possible with the VS Code extension or Zed extension,  but for an easier experience,
```

Lately, I have been using Zed because the VScode extension (the pre-release one) doesn't work properly.